scripts/release_support/changelog.sh

```diff
@@ -125,9 +125,9 @@ do
                                 pr_url=$(curl -s -H "Accept: application/json" -H "Authorization: token $ACCESS_TOKEN" https://api.github.com/repos/seagate/"$repo_name"/commits/"$commit"/pulls | jq '.[].html_url' | sed "s/\"//g")
                                 pr_number=$(echo "$pr_url" | awk -F[/] '{print $NF}')
                                 if [ "$pr_number" ] && [ "$pr_url" ]; then
-                                        echo -e "$filtered_commit_message [$pr_number]($pr_url)\n" >> $report_file
+                                        echo -e "$filtered_commit_message [$pr_url]($pr_url)" >> $report_file
                                 else
-                                        echo -e "$filtered_commit_message\n" >> $report_file
+                                        echo -e "$filtered_commit_message" >> $report_file
```

Comment:
We need to list messages without PR and work with that component team to identify the cause.

Reply:
Sure